www-cronrobot/app/views/home/index.html.erb

```diff
@@ -1 +1,176 @@
-welcome to CronRobot
\ No newline at end of file
+<section id="home">
+
+    <!-- The Navbar of the website, will contain the logo, and the links -->
+    <nav>
+      <a href="#home">
+        <img id="logo" alt="Logo" src="images/Big Logo.svg">
+      </a>
+      <div id="center-nav">
+        <a href="#features">Features</a>
+        <a href="#pricing">Pricing</a>
+        <a href="#faq">FAQ</a>
+        <a href="https://docs.cronrobot.io">Docs</a>
+      </div>
+      <a onclick="handleLogin()" class="center" id="dashboard">
+        <img src="images/icons/dashboard.svg">
+        <p class="text-medium">Dashboard</p>
+      </a>
+      <button class="openbtn" onclick="openNav()">☰</button>
+    </nav>
+
+    <h1 id="heading">
+      The world's leading uptime monitoring service<blue>.</blue>
+    </h1>
+
+    <p id="tagline">
+      Downtime happens even to the best of us. But it’s important to know it before customers are affected!
+    </p>
+
+    <a onclick="handleLogin()">
+      <button class="btn-start">
+        <p>GET STARTED NOW</p>
+        <img src="images/icons/arrow-right.svg">
+      </button>
+    </a>
+
+</section>
+
+<section id="features">
+    <h1 class="sub-heading">Features</h1>
+    <div id="features-container">
+
+      <div class="feature">
+        <div class="feature-icon-parent center" id="cron">
+          <img class="feature-icon" src="images/icons/monitor.svg">
+        </div>
+        <div class="feature-details">
+          <p class="feature-title">Cron-Job Monitoring</p>
+          <p class="feature-tagline">Also known as heartbeat monitoring. Monitor recurring background jobs.</p>
+        </div>
+      </div>
+
```

Comment:
Improve the spacing here

![image](https://user-images.githubusercontent.com/8007722/130610382-a587baa3-d0f3-41ec-a641-9bf10cab7aba.png)

Reply:
Alright

Reply:
Done 👍